GOVERNANCE.md

```diff
@@ -0,0 +1,129 @@
+
+We essentially define them as **people who actively participate in the
+project**. Examples of things that would _not_ make you a contributor are:
+
+- Submitting a single bug report and never returning
+- Writing blog posts or other evangelism
+- Using the software in production
+- Forking the project and maintaining your own version
+- Writing a third-party tool or add-on
+
+While these are all generally quite valuable, we don't consider these
+ongoing contributions to the codebase or project itself, and on their own do not
+constitute "active participation".
+
+Contributors and their input are valued and their input is expected to be taken
+and respected by Maintainers; not every discussion will result in a change, but
+it is intended that every voice will be heard.
+
+## Processes
+
+For the purposes of making decisions across the project, the following processes
+are defined.
+
+### Decision-Making
+
+The person proposing a decision to be made (i.e. technical, project direction,
+etc.) can offer a proposal, along with a 2-to-4 week deadline for discussion.
+During this time, Maintainers may participate with a vote of:
+
+A) Support B) Reject C) Abstain
+
+Each Maintainer gets one vote. The total number of "participating votes" is the
+number of Maintainer votes which are not Abstain. The proposal is accepted when
+more than half of the participating votes are Support.
+
+In the event that a decision is reached before the proposed timeline, said
+proposal can move on and be accepted immediately. In the event no consensus is
+reached, a proposal may be re-submitted later on.
+
+This document itself is subject to the Decision-Making process by the existing
+set of Maintainers.
+
+### Adding and Removing Maintainers
+
+An active Contributor may, at any given time, nominate themselves or another
+Contributor to become a Maintainer. This process is purely optional and no
+Contributor is expected to do so; however, self-nomination is encouraged for
+active participants. A vote and discussion by the existing Maintainers will be
+used to decide the outcome.
+
+Note that Contributors should demonstrate a high standard of continuous
+participation to become a Maintainer; the upper limit on the number of
+Maintainers is practically bounded, and so rejection should be considered as a
+real possibility. As the scope of the project changes, this limit may increase,
+but it is fundamentally fluid. (If you are unsure, you are free to privately ask
+existing Maintainers before self-nominating if there is room.)
+
+A Maintainer may, at any time, cede their responsibility and step down without a
+vote.
+
+A Maintainer can be removed by other Maintainers, subject to a vote of at-least
+a 2/3rds majority from the existing Maintainer group (excluding the vote of the
+Maintainer in question). This can be due to lack of participation, conduct
+violations, etc. Note that Maintainers are subject to a higher set of behavioral
```

Comment:
```suggestion
Maintainer in question). This can be due to lack of participation or conduct violations. Note that Maintainers are subject to a higher set of behavioral
```

Reply:
With this change, it seems to explicitly limit the reasons to remove a Maintainer to the two listed here, and not allow for any others. Is that intentional, or was this just meant to improve the wording? I imagine there could be some other legitimate reason, like lack of alignment with the project goals, or something like that.

Reply:
No, I just found the use of `etc.` as both an abbreviation and to end the sentence confusing, so I tried to edit it out

Reply:
IMO having the list open-ended is a little clearer than having it closed-ended - maybe "lack of participation or conduct violations, among other things"?